src/homework6/homework6.sql

```diff
@@ -0,0 +1,54 @@
+USE baseball;
```

Comment:
overall, good functioning script with all the stuff needed. Indexes seem to be correctly done, and that helps with the speed for sure!